src/audio/base_fw.c

```diff
@@ -394,6 +463,10 @@ static int basefw_set_large_config(struct comp_dev *dev,
 						last_block, data_offset, data);
 	case IPC4_ENABLE_LOGS:
 		return ipc4_logging_enable_logs(first_block, last_block, data_offset, data);
+	case IPC4_REGISTER_KCPS:
+		return basefw_register_kcps(first_block, last_block, data_offset, data);
+	case IPC4_RESOURCE_ALLOCATION_REQUEST:
```

Comment:
@kfrydryx did you try to clarify with ACE FW/Windows SW why do we have two IPC's doing actually the same?

Reply:
I did not. register_kcps is not used in internal testing, though the call is implemented. Both are handled by closed source fw.

Reply:
is 1 mcps == 1000 kcps？

Reply:
yes, exactly. Just like in metric system. 1 MCPS = 1000 KCPS = 1000 000 CPS.